Cargo.toml

```diff
@@ -19,23 +19,20 @@ resolver = "2"
 version = "0.12.0"
 authors = ["thin-edge.io team <info@thin-edge.io>"]
 edition = "2021"
-rust-version = "1.67"
+rust-version = "1.70"
 license = "Apache-2.0"
 homepage = "https://thin-edge.io"
 repository = "https://github.com/thin-edge/thin-edge.io"
 
 [workspace.dependencies]
-anyhow =  "1.0"
+anyhow = "1.0"
 assert-json-diff = "2.0"
 assert_cmd = "2.0"
 assert_matches = "1.5"
 async-compat = "0.2.1"
 async-log = "2.0"
 async-trait = "0.1"
-async-tungstenite = { version = "0.18.0", features = [
-    "tokio-runtime",
-    "tokio-rustls-native-certs",
```

Comment:
Removing the feature `tokio-rustls-native-certs` might be a source of confusion as a certificate can be trusted by all the thin-edge component expect the remote access plugin (which depends on `async-tungstenite` for web-sockets). See https://github.com/thin-edge/thin-edge.io/issues/1678

Reply:
Just confirmed that with this change, `c8y-remote-access-plugin` fails when accepting a connection from Cumulocity, with the following message in MQTT:

```text
$ tedge mqtt sub "#"

...

c8y/s/us] 502,c8y_RemoteAccessConnect,"

Error:   × Connecting to Websocket
  ├─▶ URL error: TLS support not compiled in
  ╰─▶ TLS support not compiled in

"
``` 

A ticket was created to create an integration test to catch similar problems in the future: #2201

Reply:
A fix was added in https://github.com/thin-edge/thin-edge.io/pull/2185/commits/0c356e396234080f9b089ebf801577309066f22a, and an issue to revert back into a regular dependency once a patched version is released was created #2202.

Reply:
> Just confirmed that with this change, c8y-remote-access-plugin fails when accepting a connection from Cumulocity, with the following message in MQTT

Thank you for going deeper. So even worse than we feared ;-)